docs/validation-rules.rst

```diff
@@ -648,6 +648,10 @@ Data type allowed for the key value. Can be one of the following names:
    * - ``string``
      - :func:`py2:basestring`
      - :class:`py3:str`
+   * - ``decimal``
```

Comment:
The type is always available as decimal module. Only decimal128 might be not available as part of bson module (which exist for all python version)